automated-comments/java/two-fer/use_ternary_operator.md

```diff
@@ -0,0 +1,2 @@
+This solution uses an if statement, but all of the conditional logic is small
+enough that we can better express it inside a ternary expression.
```

Comment:
Maybe include a link to a page explaining what a ternary expression is? That concept might not be known to everyone.

Reply:
Added one to https://www.baeldung.com/java-ternary-operator (looked like a pretty decent tutorial on the ternary operator.